lib/src/main/res/drawable/check.xml

```diff
@@ -0,0 +1,5 @@
+<vector android:height="24dp" android:tint="#FFFFFF"
+    android:viewportHeight="24.0" android:viewportWidth="24.0"
+    android:width="24dp" xmlns:android="http://schemas.android.com/apk/res/android">
+    <path android:fillColor="#FF000000" android:pathData="M9,16.17L4.83,12l-1.42,1.41L9,19 21,7l-1.41,-1.41z"/>
```

Comment:
We cannot use hard coded strings and values, they should come from the context theme and customizable where possible (the activity may have a dark theme and black icon/text will be invisible).

This is true for all the other resources across the library.